### PR TITLE
016/default

### PR DESCRIPTION
**Pull request by payne-chris-r**

### CODE REVIEW

**Comment by payne-chris-r** [comment]
Not sure if the auth service should be in there as blank, but I thought it'd give us a place to start...

File: app/app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ Ember.MODEL_FACTORY_INJECTIONS = true;
 App = Ember.Application.extend({
   modulePrefix: config.modulePrefix,
   podModulePrefix: config.podModulePrefix,
-  Resolver,
```
Comment (raq929) [inline app/app.js:13]: Why are we deleting trailing commas in js?

Comment (payne-chris-r) [reply app/app.js:13]: I think that's my mistake. js-standard doesn't like them, and I'm in that weird half-way stage of implementing some js-standard stuff, and not others.

**Comment by raq929** [comment]
There's a failing test on this PR having to do with the application serializer. The serializer doesn't actually DO anything, so I'm not sure how much it matters? But I think it should pass or be deleted.

The test passes on master.

**Comment by raq929** [comment]
I didn't mean to close, just comment.

**Comment by raq929** [comment]
Also, I think we should probably deal with these warnings, although I guess that's updating node globally.
```
DEPRECATION: Node v5.10.1 is no longer supported by Ember CLI. We recommend that you use the most-recent "Active LTS" version of Node.js.
WARNING: Node v5.10.1 is not tested against Ember CLI on your platform. We recommend that you use the most-recent "Active LTS" version of Node.js.
```

**Comment by payne-chris-r** [comment]
Is this the one you're talking to?
```
Promise rejected before "it serializes records": Attempting to register an unknown factory: 'model:application'
```
?

**Comment by raq929** [comment]
That's the one.

**Comment by raq929** [comment]
There's no application model. So I think we can safely delete the application serializer and this test. Because I lied. On master, this test doesn't exist.

**Comment by payne-chris-r** [comment]
It's possible that I didn't create the application serializer correctly. I'm gonna take a look.

**Comment by payne-chris-r** [comment]
I feel like `application` shouldn't have a model, but maybe it's showing that there's an issue with how `active-model-serializer` is being brought in?

**Comment by raq929** [comment]
Right, the serializer expects a model. Otherwise what would it serialize? So I think we can kill it?

**Comment by payne-chris-r** [comment]
Sounds like we're trying to extend an adapter that doesn't exist... I'm punting on this because it sounds like an issue with the version of ember-data we're using, not with the current update (see my soon-to-be-pushed branch). As you'll see my vote on this is to leave #23 open and continue doing what I was trying to do (de-podify), and come back to this later (if necessary) with 23.

**Pull request merged**